### PR TITLE
new post: add PyCon TW 2024 call for proposal

### PR DESCRIPTION
**Pull request by Anna0118**
## Types of changes
* post update

## Description

1.  Add new post: PyCon TW 2024 call for proposals
2.  The post is available in Chinese and English

Please help me check the content. Thanks a lot.

### CODE REVIEW

File: content/posts/2024-02-28-pycon-taiwan-2024-cfp-is-open.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Category: programs
+Tags: Calls for Proposals, PyCon TW 2024, legacy-blogger
+Slug: 2024-02-28-pycon-taiwan-2024-cfp-is-open!
+Authors: program team
+Summary: ![PyCon TW 2024 Call for Proposal](images/2024-02-25-pycon-tw-2024-programs/0225_CFP en.png){width=500}
+[TOC]
+
+---
+
+<!--your content here-->
```
Comment (winniepopu) [inline content/posts/2024-02-28-pycon-taiwan-2024-cfp-is-open.md:13]: I think this line can just be removed.

**Review by winniepopu — CHANGES_REQUESTED** [review]
Hi Anna,

I've reviewed the content, and it's excellent!
I have some suggestions for this post:

1. Adding a text summary would make it more attractive, especially if it includes images and text on the homepage.
2. It would be better to also add the picture to the article page.
3. The picture has some flaws Orz. I've generated a new version. Please replace it with the new one: https://drive.google.com/drive/u/1/folders/13iX5OzJngtc8DISPSg7584lD593jIqWu.

I think the article is awesome! Thanks for your contribution!

**Review by winniepopu — APPROVED** [review]
Hi Anna, yeah it looks great! Thank you!
BTW, I think it would look better to not to restrict the picture with width = 500 in the article page haha~

**Comment by Anna0118** [comment]
> Hi Anna, yeah it looks great! Thank you! BTW, I think it would look better to not to restrict the picture with width = 500 in the article page haha~

Thanks a ton for the comment. And yes, you're absolutely right... Next time, no width restrictions necessary!

**Pull request merged**